cmd/stolon-pgbouncer/main.go

```diff
@@ -186,6 +189,27 @@ func main() {
 	}()
 
 	switch command {
+	case healthcheck.FullCommand():
+		logger.Log("event", "healthcheck", "msg", "health checking local pauser API")
```

Comment:
Thoughts on doing this here vs adding a modifier like `--health-check-only` to the `failover` command that uses the same discovery mechanism as the failover itself to find its clients?

Reply:
I was thinking that could be a followup if this command proves useful.

Reply:
More broadly, I wonder if a `stolon-pgbouncer status` command would be useful - to give you an overview of all the components, their IPs and their state.

Reply:
That is probably a nicer flow, and more consistent with the stolon toolchain. I like that idea a lot actually